types/react-document-meta/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for react-document-meta 3.0
+// Project: https://github.com/kodyl/react-document-meta#readme
+// Definitions by: ulrichb <https://github.com/ulrichb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
The lower the TS version, the easier the typings are to consume.  Since these typings don't appear to be using any 2.8 features, it might be preferable to use a lower version number.

Reply:
@amcasey 

I've tried to use 2.3 but build says no :) (the reason is that the `react` dependency already requires 2.8). => Have to stay at 2.8.

Reply:
Thanks for checking!